### PR TITLE
Fix meta.json titles for Deprecated Resources

### PR DESCRIPTION
**Pull request by EliuX**
Make title of `meta.json` more verbose in the already migrated pages of **Deprecated Resources**

### CODE REVIEW

**Comment by EliuX** [comment]
Check it @mcelicalderon

File: rest/notification/instance-delete-examples/meta.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "title": "instance-delete-examples",
+  "title": "Delete notification",
```
Comment (mcelicalderon) [inline rest/notification/instance-delete-examples/meta.json]: Lets use title case in the titles, I'll mark all the errors in the PR so you can see them gone after the next commit

File: rest/notification/instance-get-example-1/meta.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-  "title": "instance-get-example-1",
+  "title": "Retrieve notification",
```
Comment (mcelicalderon) [inline rest/notification/instance-get-example-1/meta.json]: title case

File: rest/notification/list-get-example-1/meta.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
 {
-  "title": "list-get-example-1",
+  "title": "Filter notifications",
```
Comment (mcelicalderon) [inline rest/notification/list-get-example-1/meta.json]: title case

File: rest/notification/list-get-example-2/meta.json [hunk 4]
```diff
@@ -1,4 +1,4 @@
 {
-  "title": "list-get-example-2",
+  "title": "Filter notifications using a param",
```
Comment (mcelicalderon) [inline rest/notification/list-get-example-2/meta.json]: title case

File: rest/notification/list-get-example-3/meta.json [hunk 5]
```diff
@@ -1,4 +1,4 @@
 {
-  "title": "list-get-example-3",
+  "title": "Filter notifications using a limit",
```
Comment (mcelicalderon) [inline rest/notification/list-get-example-3/meta.json]: title case

File: rest/notification/list-get-example-4/meta.json [hunk 6]
```diff
@@ -1,4 +1,4 @@
 {
-  "title": "list-get-example-4",
+  "title": "Filter notifications by a range",
```
Comment (mcelicalderon) [inline rest/notification/list-get-example-4/meta.json]: title case

File: rest/sms/instance-get-example-1/meta.json [hunk 7]
```diff
@@ -1,4 +1,4 @@
 {
-  "title": "instance-get-example-1",
+  "title": "Retrieve a SMS",
```
Comment (mcelicalderon) [inline rest/sms/instance-get-example-1/meta.json]: `Retrieve an SMS`

File: rest/sms/list-get-example-1/meta.json [hunk 8]
```diff
@@ -1,4 +1,4 @@
 {
-  "title": "list-get-example-1",
+  "title": "List all SMSs",
```
Comment (mcelicalderon) [inline rest/sms/list-get-example-1/meta.json]: Title case

**Comment by EliuX** [comment]
LGTM :ship:

**Pull request merged**